src/clj/witan/send/main.clj

```diff
@@ -79,4 +79,8 @@
   ([config-path]
    (-> config-path
        read-config
-       run-recorded-send)))
+       (run-recorded-send true)))
```

Comment:
can't we just add `:print-warnings?` to the config and assoc it onto config here if we want rather than adding an arity with a default of true?

Reply:
The default is currently true. I found this provided me with the most flexibility

Reply:
OK